pipeline_annotate.sh

```diff
@@ -403,14 +409,20 @@ executeAnnotSv() {
 
   args=()
   args+=("-SVinputFile" "${inputFilePath}")
-  args+=("-outputFile" "${outputFilePath}")
+  args+=("-outputFile" "${outputFilePath}.tsv")
```

Comment:
add the .tsv in the call to the output so that the caller known the name of the output

Reply:
not sure what you mean

Reply:
```
currentOutputFilePath="${currentOutputDir}/${outputFilename}.tsv"
mkdir -p "${currentOutputDir}"
executeAnnotSv "${currentInputFilePath}" "${currentOutputFilePath}" "${assembly}" "${phenotypes}"
```

```
args+=("-outputFile" "${outputFilePath}")
```